graphql/schema.graphql

```diff
@@ -93,19 +97,35 @@ type Mutation {
     paymentTerms: [PaymentTermInput]
     priceTables: [String]
   ): MutationResponse
+    @withSession
+    @withPermissions
```

Comment:
Only store admins should be able to update an organization, so these directives can be removed and replaced with a new `@checkAdminAccess` directive that checks the `adminUserAuthToken`

Same with `createOrganization` above and `deleteOrganization` below

Reply:
Additionally, `@withPermissions` doesn't actually control access, it just gives the resolver access to permission data. There are many mutations (for example all of the Cost Center mutations) where the app is not checking if the user has permission to do that particular operation.

For these cases where the mutation can be called by either a store user or admin user, we can create a directive that checks for either a `adminUserAuthToken` or a `storeUserAuthToken` and validates whichever one is present. Maybe `@checkUserAccess`, what do you think?

Then we should still add some additional validation in each resolver to make sure (for example) the user has access to the organization they're trying to create a cost center in.